.github/ISSUE_TEMPLATE/release-checlist.md

```diff
@@ -0,0 +1,36 @@
+---
+name: Release Checklist
+about: Template issue for creating release checklists.
+title: Checklist for vX.Y.Z release
```

Comment:
Can be automated in a workflow this way by settings a VERSION environment variable

```suggestion
title: Checklist for {{ env.VERSION }} release
```